configs/nemtech.json

```diff
@@ -2,15 +2,20 @@
   "index_name": "nemtech",
   "start_urls": [
     {
-      "url": "https://nemtech.github.io/nem2",
-      "tags": [
-        "nem2"
-      ],
-      "selectors_key": "nem2"
+      "url": "https://nemtech.github.io/(?P<lang>.*?)/",
```

Comment:
Could you set the `lang` attribute from the main `<htm/>` element within your DOM instead ?
You can find some [explanation here](https://www.w3.org/International/questions/qa-html-language-declarations).